src/Embed.js

```diff
@@ -161,9 +162,18 @@ export const Initialize = async ({client, target, url, playerOptions, setPageTit
         "asset_metadata/title",
         "asset_metadata/display_title",
         "asset_metadata/nft"
-      ]
+      ],
+      produceLinkUrls: true
     })) || {};
 
+    if(metadata.asset_metadata?.nft?.media_type === "Ebook") {
```

Comment:
is this ?. format supported now? I'm gonna start using that all the time

Reply:
Well it's not supported by IE https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining